src/components/layermanager/style/layermanager.less

```diff
@@ -190,9 +189,11 @@
   }
 
   .ga-layer-folded {
+    height: 3em;
 
     .ga-layer-tools {
-      margin-left: 100%;
+      top: 0em;
+      z-index: -1;
```

Comment:
This `z-index` doesn't work as expected the element is behind the `<li>`parent. Remove it. And put 2 different z-index to `.ga-layer-tools` and `.ga-layer-infos`